ios/iNaturalistReactNative.xcodeproj/xcshareddata/xcschemes/iNaturalistReactNative.xcscheme

```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
Is this something we're going to want to revert when actually releasing?

Reply:
hmm, this file changes automatically for me when I build with XCode

Reply:
Ok, I'm going to assume that's not a big deal then, but something to keep in mind if we have problems with the next build.